src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/events/SupplementaryEvidenceWithOcrTest.java

```diff
@@ -0,0 +1,86 @@
+import com.microsoft.azure.servicebus.Message;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.SpyBean;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.config.Environment;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.config.IntegrationTest;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.domains.envelopes.EnvelopeEventProcessor;
+
+import java.util.concurrent.TimeUnit;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.givenThat;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.postRequestedFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static org.awaitility.Awaitility.await;
+import static org.mockito.BDDMockito.given;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData.fileContentAsString;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.config.Environment.CASE_SEARCH_URL;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.config.Environment.CASE_TYPE_BULK_SCAN;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.config.Environment.CASE_TYPE_EXCEPTION_RECORD;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.config.Environment.EXCEPTION_RECORD_SEARCH_URL;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.config.Environment.GET_CASE_URL;
+
+@IntegrationTest
+class SupplementaryEvidenceWithOcrTest {
+
+    private static final String CASE_EVENT_TRIGGER_START_URL = Environment.CASE_EVENT_TRIGGER_START_URL
+        .replace(CASE_TYPE_BULK_SCAN, CASE_TYPE_EXCEPTION_RECORD);
+    private static final String CASE_SUBMIT_URL = Environment.CASE_SUBMIT_URL
+        .replace(CASE_TYPE_BULK_SCAN, CASE_TYPE_EXCEPTION_RECORD);
+
+    private static final String ELASTICSEARCH_EMPTY_RESPONSE = "{\"total\": 0,\"cases\": []}";
+
+    @SpyBean
+    private IMessageReceiver messageReceiver;
+
+    @Autowired
+    private EnvelopeEventProcessor envelopeEventProcessor;
+
+    @BeforeEach
+    void before() {
+        givenThat(get(GET_CASE_URL).willReturn(aResponse().withStatus(HttpStatus.NOT_FOUND.value())));
+
+        givenThat(get(CASE_EVENT_TRIGGER_START_URL).willReturn(aResponse().withBody(
+            "{\"case_details\":null,\"event_id\":\"eid\",\"token\":\"etoken\"}"
+        )));
+
+        givenThat(post(CASE_SEARCH_URL).willReturn(
+            aResponse().withBody(ELASTICSEARCH_EMPTY_RESPONSE)
+        ));
+
+        givenThat(post(EXCEPTION_RECORD_SEARCH_URL).willReturn(
+            aResponse().withBody(ELASTICSEARCH_EMPTY_RESPONSE)
+        ));
```

Comment:
Will this test suite contain more than 1 test case? Then keeping in before setup makes sense otherwise perhaps we can just move to separate setup method and invoke in test case?

Reply:
I'll move it to the existing class `ExceptionRecordCreationTest`, where we have all exception record related tests